tlscanary/template/js/log_handler.js

```diff
@@ -0,0 +1,75 @@
+
+var type_map = {};
+var visible_columns = {};
+var final_data = {};
+var transform_rules;
+
+function test(arg)
+{
+	alert("Hello world " + arg);
+}
+
+function find_prop (obj, prop, defval)
+{
+	if (defval == undefined) defval = null;
```

Comment:
Not sure if you are aware of the difference between lax `==` and strict `===`, but this is a place that probably should use strict comparison. Also, the `typeof foo === 'undefined'` you used three lines down is usually the better test for undefined.

It might make sense to revisit all new JS code and use `==` and `===` as intended. Most times you'll want strict comparison.

Reply:
Indeed, strict equality/inequality operators make most sense for string comparison. I am making these changes as requested.